R/describe_distribution.R

```diff
@@ -34,14 +44,96 @@
 #' data(iris)
 #' describe_distribution(iris)
 #' describe_distribution(iris, include_factors = TRUE, quartiles = TRUE)
+#' describe_distribution(list(mtcars$mpg, mtcars$cyl))
 #' @export
 describe_distribution <- function(x, ...) {
   UseMethod("describe_distribution")
 }
 
+#' @rdname describe_distribution
+#' @export
+
+describe_distribution.list <- function(x,
+                                       centrality = "mean",
+                                       dispersion = TRUE,
+                                       iqr = TRUE,
+                                       range = TRUE,
+                                       quartiles = FALSE,
+                                       ci = NULL,
+                                       include_factors = FALSE,
+                                       iterations = 100,
+                                       threshold = .1,
+                                       verbose = TRUE,
+                                       ...) {
+
+  factor_el <- which(sapply(x, is.factor))
+  num_el <- which(sapply(x, is.numeric))
+
+  # get elements names as is
+  # ex: list(mtcars$mpg, mtcars$cyl) -> c("mtcars$mpg", "mtcars$cyl")
+  nm <- sapply(sys.call()[[2]], deparse)[-1]
```

Comment:
I suggest using `.safe_deparse` instead of `deparse` here.

Reply:
@etiennebacher In case you wish to check out its definition, you can find it here:

https://github.com/easystats/datawizard/blob/2a4e53ba1f64c70386a4c8f704d4a46bbe6a81e0/R/demean.R#L415-L418